WordPress/UITestsFoundation/BaseScreen.swift

```diff
@@ -0,0 +1,113 @@
+import XCTest
+
+// Access control `open` so types outside of this framework can subclass it, which is exactly how
+// this type is meant to be used.
+open class BaseScreen {
```

Comment:
This is extracted from the WordPressUITests target. It was actually added to multiple targets. 

One extra benefit of moving logic shared between targets in a dedicated framework is that it gets a clear home and you don't need to _remember_ to add new files to all targets, because the build system manages framework linking for you 😄